src/disco/topo/fd_topo.h

```diff
@@ -167,6 +167,7 @@ typedef struct {
 
     struct {
       ulong tcache_depth;
+      ulong hashmap_seed;
```

Comment:
Not here, just in the ctx with privileged_init

Reply:
fixed